3_visualize/src/plot_site_data.R

```diff
@@ -7,7 +7,7 @@ plot_site_data <- function(out_file, site_data, parameter) {
     geom_line() +
     geom_point(data=filter(site_data, !(Quality %in% c('A','P'))), size=0.1) +
     ylab(dataRetrieval::parameterCdFile %>% filter(parameter_cd == parameter) %>% pull(parameter_nm)) +
-    ggtitle(site_data$Site[1])
+    ggtitle(paste0("site_no:", site_data$Site[1], ", sate:", site_data$State[1]))
```

Comment:
teeny typo that doesn't really matter anyways but I will comment! `sate` >> `state`